shared/components/ConferenceJobs/index.js

```diff
@@ -4,7 +4,7 @@ import JobsSection from '../JobsSection';
 
 const isEmpty = array => !Array.isArray(array) || array.length === 0;
 
-const hasNoJobs = array => array.every(partner => isEmpty(partner.jobs));
+const hasNoJobs = array => Array.isArray(array) && array.every(partner => isEmpty(partner.jobs));
```

Comment:
@zoeabryant This will return false if the array is undefined. It should be true.